kustomize/commands/localize/flagscope.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package localize
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+)
+
+func AddFlagScope(allocatedFlags *flags, set *pflag.FlagSet) {
+	// no shorthand to avoid conflation with other flags
+	set.StringVar(&allocatedFlags.scope,
```

Comment:
Creating a whole separate file/function for a single flag seems unnecessary to me, It might make more sense to do this inline.

Reply:
Done.